src/app/app.component.ts

```diff
@@ -1,10 +1,50 @@
 import { Component } from '@angular/core';
 
+class Point {
+  x: number;
+  y: number;
+
+  constructor(xNum: number, yNum: number) {
+    this.x = xNum;
+    this.y = yNum;
+  }
+  add(newPoint: Point){
+    return newPoint = new Point(this.x, + this.y);
```

Comment:
this will create a new Point with same values as the existing Point. The requirement is to create a new Point with the values of x and y to be the sum of existing point's x's value and newPoint's x's value and a similar calculation for y.